Api/Modpacks.cs

```diff
@@ -28,7 +28,8 @@ static class Modpacks {
         public static void AddRowButton_Click(object sender, EventArgs e) {
             OpenFileDialog ofd = new OpenFileDialog {
                 InitialDirectory = "::{20D04FE0-3AEA-1069-A2D8-08002B30309D}",  // using the GUID to access 'This PC' folder
-                Multiselect = true
+                Multiselect = true,
+                Title = "Select a new mod file to apply to your game:"
```

Comment:
Users can select multiple files in this dialog, but also the create modpack tab doesn't apply anything to the game. I think it's best to reword this to: "Select one or more mod files"

Reply:
Sounds good, thanks for that I didn't realize. Done